mediapicker/src/main/java/org/wordpress/android/mediapicker/ui/MediaPickerFragment.kt

```diff
@@ -277,6 +281,18 @@ internal class MediaPickerFragment : Fragment(), MenuProvider {
         }
     }
 
+    private fun applyEdgeToEdgeSettings(binding: MediaPickerLibFragmentBinding){
+        binding.root.doOnApplyWindowInsets(
+            insetsMask = WindowInsetsCompat.Type.navigationBars(),
+            consumeInsets = true
+        ) { insets ->
+            binding.recycler.addItemDecoration(LastItemPaddingDecoration(insets.bottom), 0)
```

Comment:
@atorresveiga 👋
Would it do the same, just slightly simpler?

```
binding.recycler.clipToPadding = false
binding.recycler.updatePadding(bottom = insets.bottom)
```

Reply:
Nice suggestion. Fixed here [732de84](https://github.com/wordpress-mobile/WordPress-MediaPicker-Android/pull/87/commits/732de847d8d8d319ffc262c6c086f2dd38e0e807)